doc/source/user_guide/general_settings.rst

```diff
@@ -8,11 +8,19 @@ and set up units using :ref:`ref_solver_tui_commands`.
 
 Checking the mesh
 -----------------
-The ``mesh.check`` TUI performs various mesh consistency checks and displays a
+The following example shows a comparison between the TUI command and the
+python code for performing various mesh consistency checks and displays a
 report in the console that lists domain extents, volume statistics,
 face area statistics, any warnings, and information about various check and mesh
-failures. THe level of information shown depends on the setting specified for
-``mesh.check-verbosity``.
```

Comment:
You seem to have lost the verbosity command.  Maybe you can break it out as a separate example and add some details about the valid verbosity levels?